configure/CMakeLists.txt

```diff
@@ -54,13 +54,25 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     message("found GNU compiler ...")
     if(CMAKE_BUILD_TYPE MATCHES RELEASE)
         message("setup for release build ...")
-        add_compile_options(-O2)
+	if(CMAKE_VERSION VERSION_GREATER 2.8.12)
```

Comment:
This is redundant CMake sets optimizations by default: [https://cmake.org/pipermail/cmake/2007-April/013544.html](https://cmake.org/pipermail/cmake/2007-April/013544.html)